code-challenges.js

```diff
@@ -1,68 +1,168 @@
+// console.log(fruity(fruit3, fruit4)) //cherry
 
-const temp3 = 212
-// Expected output: "212 is at boiling point"
 
-// --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-// Describe your process:
+// // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
+// const temp1 = 42
+// // Expected output: "42 is below boiling point"
 
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
+// const temp2 = 350
+// // Expected output: "350 is above boiling point"
 
-// Describe your process:
+// const temp3 = 212
+// // Expected output: "212 is at boiling point"
 
-const currentCohort = "Golf 2022"
+// // Describe your process:
+
+// 1. Create a function called water
+// 2. Input a number to represent the temperature of the water and a number to represent the boiling point of water
+// 3. The function should compare the temperature of the water against boiling point of water
+// 4. Output should be a string that says "The water is below, at, or above boiling point"
+// 5. Use a conditional to make an evaluation on the temperature of the water
+
+// if the temperature of the water evaluates to below 212 degrees, it's below boiling point
+// if the temperature of the water evaluates to equal to 212 degrees, it's at boiling point
+// if the temperature of the water evaluates to above 212 degrees, it's above boiling point
+// we need an else if none of the above is true, it will return an error
+
+// const water = (waterTemp) => {
+//     if(waterTemp === 212) {
+//         return "212 is at boiling point"
+//     } else if(waterTemp > 212) {
+//         return "350 is above boiling point"
+//     } else if(waterTemp < 212) {
+//         return "42 is below boiling point"
+//     } else {
+//         return "Sorry that is not a valid temp"
+//     }
+// }
+
+// console.log(water(212))
+// console.log(water(350))
+// console.log(water(42))
+// console.log(water("test"))
+
+
+
+
+// // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
+
+
+// // Describe your process:
+// 1. Create a variable called padresRunsarray that will combine the values of two arrays then return the length
+// 2. The function uses the .length property
+// 3. The .length property looks at the index then returns the number of values in the array
+
+
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// // Expected output: 9
+
+// var padresRunsArray = [2, 5, 2, 2, 4, 6, 3, 5, 3,]
```

Comment:
Try using built in methods to combine the padres1984WorldSeriesRuns and padres1998WorldSeriesRun variables together.